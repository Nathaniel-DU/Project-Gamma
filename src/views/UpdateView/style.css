button {
    width: 50%;
    margin-top: 50px;
    display: block;
    margin: auto;
    margin-bottom: 20px!important;
}

h1 {
    color: #ADEFD1FF;
    /* float: right; */
    
}

.button-list {
    margin-top: 200px;
    
} 

