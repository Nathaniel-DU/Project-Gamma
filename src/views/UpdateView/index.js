import React, { Fragment } from "react";
import StartLocationButton from "../../components/StartLocationButton"
import './style.css';

const StartLocation = () => (
  
  <Fragment>
    <h1>StaySafe</h1>
    <div className="button-list">
      <StartLocationButton className="stop-btn" text='Stop'></StartLocationButton>
      <StartLocationButton className="excuse-btn" text='Excuse Call'></StartLocationButton>
      <StartLocationButton className="pickup-btn" text='Ask for Pickup'></StartLocationButton>
      <StartLocationButton className="panic-btn" text='Request Emergency Assistance'></StartLocationButton>
      <StartLocationButton className="update-loc-btn" text='Update My Location'></StartLocationButton>
    </div>
    
  </Fragment>
);

export default StartLocation;