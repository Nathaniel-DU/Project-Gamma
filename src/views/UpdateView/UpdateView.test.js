import React from 'react';
import { shallow } from 'enzyme';
import UpdateView from './index';

describe(`UpdateView`, () => {
    it(`should render correctly in "debug" mode`, () => {
        const component = shallow(<UpdateView debug />);
        expect(component).toMatchSnapshot();
    });
});