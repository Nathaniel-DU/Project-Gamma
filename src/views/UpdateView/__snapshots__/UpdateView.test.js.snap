// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateView should render correctly in "debug" mode 1`] = `
<Fragment>
  <h1>
    StaySafe
  </h1>
  <div
    className="button-list"
  >
    <ModalPage />
  </div>
</Fragment>
`;
